### PR TITLE
Brazilian portuguese translation

### PR DESCRIPTION
**Pull request by paulo-roger**
Added the brazilian portuguese translation to the add-on following the guides from mozilla.
translated also the description, permissions explanation, privacy note, the most recent changelog (html) and the next.md.

### CODE REVIEW

**Comment by rugk** [comment]
Hi @paulo-roger,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Please subscribe to #40  to get notified for new updates on translations etc. :smiley:

Also note you can (automatically) [let issues close](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) when a PR is merged by adding some "magic" text to your PR _body_. (Manually linked it now.)

**Review by rugk — COMMENTED** [review]
Do you think it would be acceptable to use your translation not only for `pt-br` , but for `pt` in general? (As we do not yet have a translation for `pt`?). 

Requesting a potential review from @CGReinhold, [offline-qr-code translator](https://github.com/rugk/offline-qr-code/blob/master/CONTRIBUTORS.md#portuguese) and @marceloaof and @joaooliva, [who translated the awesome emoji picker](https://github.com/rugk/awesome-emoji-picker/blob/master/CONTRIBUTORS.md#portuguese). Unless there are objections, this will be merged soon.

File: assets/texts/pt-br/Changelog/1.2.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<ul>
```
Comment (rugk) [inline assets/texts/pt-br/Changelog/1.2.html:1]: You don't need to include files you did not translate, so feel free to delete these untranslated files here.

File: assets/texts/pt-br/Changelog/1.3.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<ul>
+    <li><strong>Destaque:</strong> agora você pode contornar o pop-up e carregar diretamente o pop-up no site existente. (<a href="https://github.com/rugk/mastodon-simplified-federation/issues/44">#44</a>)</li>
+    <li><strong>Novo:</strong> o complemento agora suporta o modo de sistema escuro na página de opções.</li>
+    <li><strong>Novo:</strong> o redirecionamento de instâncias do Friendica agora é suportado. (<a href="https://github.com/rugk/mastodon-simplified-federation/issues/20">#20</a>)</li>
+    <li><strong>Novo:</strong> a tradução para espanhol foi adicionada, graças a <a href="https://github.com/dbarrerap">@dbarrerap</a>.</li>
+    <li><strong>Novo:</strong> a tradução para francês foi adicionada, graças a <a href="https://github.com/PoorPocketsMcNewHold">@PoorPocketsMcNewHold</a>.</li>
+    <li><strong>Novo:</strong> a tradução para russo foi adicionada, graças a <a href="https://github.com/katabasiz">@katabasiz</a>.</li>
+    <li><strong>Melhoria:</strong> a tradução para turco foi atualizada, graças a <a href="https://github.com/hozansahin">@hozansahin</a>.</li>
+    </ul>
```
Comment (rugk) [inline assets/texts/pt-br/Changelog/1.3.html:9]:
```suggestion
</ul>
```

File: assets/texts/pt-br/Changelog/1.3.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    
+    Mais informações <a href="https://github.com/rugk/mastodon-simplified-federation/releases/tag/v1.3">no GitHub</a>.
```
Comment (rugk) [inline assets/texts/pt-br/Changelog/1.3.html:11]:
```suggestion
Mais informações <a href="https://github.com/rugk/mastodon-simplified-federation/releases/tag/v1.3">no GitHub</a>.
```

File: src/_locales/pt-br/messages.json [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+  "translatorCredit": {
+    "message": "Este complemento foi traduzido para o inglês por $TRANSLATORS$.",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://pauloroger.dev/",
```
Comment (rugk) [inline src/_locales/pt-br/messages.json:193]: Okay fair enough, link wherever you want. :slightly_frowning_face:

**Review by rugk — APPROVED** [review]

File: assets/texts/pt-br/Changelog/next.md [hunk 5]
```diff
@@ -0,0 +1 @@
+* **Corrigido:** Foi corrigido um bug que fazia com que o redirecionamento não fosse capturado no caso de versões mais recentes do Friendica. ([#62](https://github.com/rugk/mastodon-simplified-federation/issues/62))
```
Comment (rugk) [inline assets/texts/pt-br/Changelog/next.md:1]: Note: Also feel free to already add the "translation added" line here (also in the English version of this file).

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit

**Comment by paulo-roger** [comment]
> Do you think it would be acceptable to use your translation not only for `pt-br` , but for `pt` in general? (As we do not yet have a translation for `pt`?).
> 
> Requesting a potential review from @CGReinhold, [offline-qr-code translator](https://github.com/rugk/offline-qr-code/blob/master/CONTRIBUTORS.md#portuguese) and @marceloaof and @joaooliva, [who translated the awesome emoji picker](https://github.com/rugk/awesome-emoji-picker/blob/master/CONTRIBUTORS.md#portuguese). Unless there are objections, this will be merged soon.

I am going to make the European Portuguese in a new PR. I have learned Portuguese people often doesn't like to use Brazilian portuguese so thats why I made the distinction, I am trying to avoid complains.

File: assets/texts/pt-br/permissions.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Permissões solicitadas
+
+Para uma explicação geral da permissão de complemento, consulte [this support article](https://support.mozilla.org/kb/permission-request-messages-firefox-extensions).
```
Comment (CGReinhold) [inline assets/texts/pt-br/permissions.md:3]:
```suggestion
Para uma explicação geral da permissão de complemento, consulte [este artigo de suporte](https://support.mozilla.org/kb/permission-request-messages-firefox-extensions).
```

File: assets/texts/pt-br/amoDescription.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+Saiba mais sobre o Mastodon em <a href="https://joinmastodon.org/">joinmastodon.org</a>.
+
+
+<b>🤠 Suporte estendido 🤠</b>
+Este complemento suporta redirecionamentos de outros sistemas no Fediverse, nomeadamente GNU Social, Friendica e Pleroma. Especialmente no primeiro caso, no entanto, o suporte pode ser um pouco limitado.
+
+
+<b>📝 Desenvolvimento 📝</b>
+O complemento é um software de código aberto gratuito/livre e desenvolvido no GitHub. <a href="https://github.com/rugk/mastodon-simplified-federation">Fork ele no GitHub</a> e contribua.
+<a href="https://github.com/rugk/mastodon-simplified-federation/contribute">Há alguns problemas simples para começar.</a> Ou você pode <a href="https://github .com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTING.md#translations">traduzir o complemento para seu próprio idioma.</a>
```
Comment (CGReinhold) [inline assets/texts/pt-br/amoDescription.html:29]:
```suggestion
<a href="https://github.com/rugk/mastodon-simplified-federation/contribute">Há alguns problemas simples para começar.</a> Ou você pode <a href="https://github.com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTING.md#translations">traduzir o complemento para seu próprio idioma.</a>
```

File: assets/texts/pt-br/amoDescription.html [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+<li>📃 <a href="https://github.com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTING.md#translations">Traduza este complemento para vários idiomas!</a></li>
+<li>🐛 <a href="https://github.com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTING.md#coding">Corrija alguns problemas simples e comece a desenvolver complementos</a> (ou apenas experimente a versão de desenvolvimento)</li>
+<li>💡 <a href="https://github.com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTING.md#need-ideas">Ou confira alguns outros problemas adicionais</a > (ou traduzi-los).</li>
+</ul>
+Ou, em qualquer caso, <a href="https://github.com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTING.md#support-us">apoie-nos divulgando!</a > ❤️
+
+
+<b>💬 Permissões 💬</b>
+Este complemento requer o mínimo de permissões possível.
+Uma explicação de todas as permissões, solicitações deste complemento, pode ser encontrada <a href="https://github.com/rugk/mastodon-simplified-federation/blob/master/assets/texts/en/permissions.md">neste site</a>.
```
Comment (CGReinhold) [inline assets/texts/pt-br/amoDescription.html:44]:
```suggestion
Uma explicação de todas as permissões, solicitações deste complemento, pode ser encontrada <a href="https://github.com/rugk/mastodon-simplified-federation/blob/master/assets/texts/pt-br/permissions.md">neste site</a>.
```

File: assets/texts/pt-br/privacy.txt [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+Para tanto, o COMPLEMENTO poderá entrar em contato com o SERVIDOR REMOTO e LOCAL SERVER e enviar informações sobre o USUÁRIO REMOTO e MENSAGEM REMOTA.
+Além disso, pode enviar informações da PRÓPRIA CONTA e PRÓPRIO SERVIDOR para PRÓPRIO SERVIDOR.
+
+O COMPLEMENTO não enviará informações sobre sua CONTA PRÓPRIA ou SERVIDOR PRÓPRIO para nenhum SERVIDOR REMOTO. Se esse caso ocorrer inadvertidamente, é considerado uma vulnerabilidade no COMPLEMENTO.
+Pode acontecer, porém, que o SERVIDOR REMOTO seja o mesmo que o SERVIDOR PRÓPRIO. Nesse caso, o COMPLEMENTO funciona da mesma forma, podendo assim enviar toda a informação para o seu próprio servidor.
+
+Uma explicação de todas as permissões, solicitações deste COMPLEMENTO, pode ser encontrada em https://github.com/rugk/mastodon-simplified-federation/blob/master/assets/texts/en/permissions.md.
```
Comment (CGReinhold) [inline assets/texts/pt-br/privacy.txt:20]:
```suggestion
Uma explicação de todas as permissões, solicitações deste COMPLEMENTO, pode ser encontrada em https://github.com/rugk/mastodon-simplified-federation/blob/master/assets/texts/pt-br/permissions.md.
```

**Review by CGReinhold — APPROVED** [review]
Great work 🥇 ! Added a couple of suggestions that could maybe be useful.

**Comment by rugk** [comment]
> I am going to make the European Portuguese in a new PR. I have learned Portuguese people often doesn't like to use Brazilian portuguese so thats why I made the distinction, I am trying to avoid complains.

Fine yeah, only you (as in all contributors that speak the language) can make that decision/know what's appropriate there.
One technical side-note from me though: AFAIK when sentences are the same, you only need to define the differences in `pt-br`, it will automatically fallback to `pt`  (and then to `en`), so you don't need to duplicate all that (though the only web-ext translator tool may not notice that…)

File: assets/texts/pt-br/Changelog/1.2.html (hunk 1 above)
Comment (paulo-roger) [reply assets/texts/pt-br/Changelog/1.2.html:1]: Thanks for the tip! I will leave as a reminder to myself to translate it later if that's OK. This is indeed an opportunity for me to work together with other people in GitHub for the 1st time, so I am learning things. Please feel free to point out best practices, I have read some stuff but I know I am probably gonna make some mistakes further on.

File: assets/texts/pt-br/Changelog/next.md (hunk 5 above)
Comment (paulo-roger) [reply assets/texts/pt-br/Changelog/next.md:1]: Will do.

File: assets/texts/pt-br/amoDescription.html (hunk 7 above)
Comment (paulo-roger) [reply assets/texts/pt-br/amoDescription.html:29]: Thanks 👍

**Comment by rugk** [comment]
As you wanted another best practice information for "How to Git/GitHub" here you go:
Next time, try to avoid doing a pull request from the master branch, because you can run into problems when you have a "non-clean" master that does not follow this repo here (i.e. "upstream"). [See this article for details](https://help.github.com/en/articles/proposing-changes-to-your-work-with-pull-requests). Anyway, this is only a tip for the next time. :smiley:

**Review by rugk — CHANGES_REQUESTED** [review]

**Comment by rugk** [comment]
@paulo-roger  If you need any pointers/help or have questions, feel free to ask.

File: assets/texts/pt-br/Changelog/next.md (hunk 5 above)
Comment (rugk) [reply assets/texts/pt-br/Changelog/next.md:1]: done by me now

**Pull request merged**